phpstan.neon.dist

```diff
@@ -0,0 +1,37 @@
+parameters:
+	tmpDir: build/phpstan
+	level: 5
```

Comment:
Codeigniter has recently been launched to level 6.

https://github.com/datamweb/CodeIgniter4/blob/bdbb77566563e1bc5c5668fcb3a58b202946e143/phpstan.neon.dist#L17

Reply:
Well... I'm afraid that level 6 will flood us with errors 😜 

Let's keep it and see how all this is going to work for us in this project.

Reply:
Ok, right now we have to say hello to the brain.